imagepaste/clipboard/darwin/darwin.py

```diff
@@ -65,7 +66,8 @@ def push(cls, save_directory: str) -> DarwinClipboard:
                 "close access pastedImage",
             ]
             process = Process.execute(cls.get_osascript_args(commands))
-            if process.stderr:
+
+            if not isfile(filepath):
                 return cls(Report(3, f"Cannot save image: {image} ({process.stderr})"))
```

Comment:
Yeah, so I think it's a good idea to make it shown:
```suggestion
            if not isfile(filepath):
                return cls(Report(3, f"Cannot save image: {image} ({process.stderr})"))
            if process.stderr:
                report = Report(6, f"Saved 1 image: {image} (WARN: {process.stderr})")
                return cls(report, [image])
```